### ISSUE TITLE
Help for predicting the learning curve of training losses

### ISSUE THREAD

**Issue by f4nku4n** [automl/nas-bench-x11#4]
Hi guys,
I'm practicing on the example code and I found that the API only supported for predicting the validation accuracy, but in your paper, I see that the API can predict for the losses. Can you make an example for using API to predict the train/validation losses.
Thank you guys for a great paper.
